File: backend/olympic_it_project/src/main/java/org/example/olympic_ot_project/dto/exam/UpdateExamRequest.java
package org.example.olympic_ot_project.dto.exam;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class UpdateExamRequest {
    @NotBlank(message = "Tên cuộc thi không được để trống")
    private String name;
    private boolean shuffleOption;
}

File: frontend/olympic_it_project/olympic_it_project/lib/dto/admin_manager/exam/update_exam_request.dart
class UpdateExamRequest {
  final String name;
  final bool shuffleOption;

  UpdateExamRequest({
    required this.name,
    required this.shuffleOption,
  });

  Map<String, dynamic> toJson() {
    return {
      'name': name,
      'shuffleOption': shuffleOption,
    };
  }
}
